pkg/plugin/state.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/neondatabase/autoscaling/pkg/util/xact"
 )
 
+const autoscalerSchedulerName = "autoscale-scheduler"
```

Comment:
We should already have this defined as a field in the config. Let's use that instead?

(see also `checkSchedulerName` in `plugin.go`)